protect_archiver/client/__init__.py

```diff
@@ -13,7 +17,7 @@ def __init__(
         port: int = Config.PORT,
         protocol: str = Config.PROTOCOL,
         username: str = Config.USERNAME,
-        password: str = Config.PASSWORD,
+        password: Optional[str] = Config.PASSWORD,
```

Comment:
I'm not sure if the password variable can ever be considered optional when interacting with a UniFi Protect Server...?

Reply:
Yes, that's because in `Config.PASSWORD` it's default to none, so it won't be correct setting the password type to `str` since we'll have to force a default of `""` which can be dangerous, so `Optional[str]` is the best type to represent the password here